includes/class-amp-story-post-type.php

```diff
@@ -1397,22 +1397,27 @@ public static function render_post_title_block( $attributes, $content ) {
 	public static function render_block_with_google_fonts( $block_content, $block ) {
 		$font_family_attribute = 'ampFontFamily';
 
-		// Short-circuit if no font family present.
 		if ( empty( $block['attrs'][ $font_family_attribute ] ) ) {
 			return $block_content;
 		}
 
-		// Short-circuit if there is no Google Font or the font is already enqueued.
 		$font = self::get_font( $block['attrs'][ $font_family_attribute ] );
-		if ( ! isset( $font['handle'], $font['src'] ) || ! $font || wp_style_is( $font['handle'] ) ) {
+
+		if ( ! $font ) {
 			return $block_content;
 		}
 
-		if ( ! wp_style_is( $font['handle'], 'registered' ) ) {
-			wp_register_style( $font['handle'], $font['src'], [], null, 'all' ); // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
+		if ( ! isset( $font['handle'], $font['src'] ) ) {
+			// @todo Prevent adding duplicates!
```

Comment:
Since `wp_add_inline_style()` is not being used, then de-duplication should be handled automatically by the style sanitizer.